cms/io/rpc.py

```diff
@@ -94,8 +94,7 @@ def __init__(self, remote_address):
 
         """
         self.remote_address = remote_address
-        self.connected = False
-        self.connected_event = gevent.event.Event()
+        self.connected = gevent.event.Event()
```

Comment:
This was a somehow public-ish api (hence all the places you had to change). Maybe a property connected that looks at _connected_event.is_set()?